client/stylesheets/_toCat.scss

```diff
@@ -4338,7 +4338,7 @@ On click the images will be displayed at normal size to complete the effect
 }
 
 .profileReceivedDecline {
-    margin-left: 10px; 
+    margin: 0 0 0 10px !important; 
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces